packages/local_auth/local_auth_platform_interface/lib/types/auth_strings.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:intl/intl.dart';
+
+/// Message shown on a button that the user can click to go to settings pages
+/// from the current dialog. It is used on both Android and iOS sides.
```

Comment:
s/. It is used on both Android and iOS sides/, on applicable platforms/

Reply:
Why does this need to be in the platform interface though? I wouldn't expect this to be handled differently than the other strings.

Reply:
I moved it here because it is the only string that is shared between both iOS and Android. The others are all platform specific and therefore have been moved to their own respective platform packages. The native packages both require this string and the platform interface is their only shared dependency, hence why it went here.

Alternatively, we could just duplicate this string to both native packages so it can be removed from the platform interface?

Reply:
Yes, we should not encode details of what happens to (currently) be true of the two platforms we happen to have implementations for into the platform interface.